autoupdate tries to add unchanged package.json

This can happen if a old version is added but `latestVersion` didn't change.
```
2020/07/02 14:32:29 a/atrament.js.json: /home/runner/work/cdnjs/cdnjs/cdnjs/ajax/libs/atrament.js/1.2.0 already exists; aborting
2020/07/02 14:32:29 a/atrament.js.json: adding version .1.1.0
2020/07/02 14:32:29 a/atrament.js.json: ignoring hidden version /home/runner/work/cdnjs/cdnjs/cdnjs/ajax/libs/atrament.js/.1.1.0
2020/07/02 14:32:29 a/atrament.js.json: run /usr/bin/git add /home/runner/work/cdnjs/cdnjs/cdnjs/ajax/libs/atrament.js/.1.1.0
2020/07/02 14:32:46 a/atrament.js.json: run /usr/bin/git commit -m Add atrament.js v.1.1.0
2020/07/02 14:33:03 a/atrament.js.json: adding latest version to package.json 1.2.0
2020/07/02 14:33:03 a/atrament.js.json: run /usr/bin/git add /home/runner/work/cdnjs/cdnjs/cdnjs/ajax/libs/atrament.js/package.json
2020/07/02 14:33:20 a/atrament.js.json: run /usr/bin/git commit -m Add atrament.js package.json (v1.2.0)
On branch master
Your branch is ahead of 'origin/master' by 22 commits.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean

panic: exit status 1

goroutine 1 [running]:
github.com/cdnjs/tools/util.CheckCmd(0xc000436c00, 0xa0, 0x600, 0x9009a0, 0xc0000a4880, 0xc0000a4880, 0xc0002f55e0)
	/home/runner/work/tools/tools/util/check.go:19 +0x137
github.com/cdnjs/tools/packages.GitCommit(0x907da0, 0xc0002ff020, 0xc0002735f0, 0x23, 0xc0004fdb30, 0x25)
	/home/runner/work/tools/tools/packages/git.go:84 +0x1c1
main.commitPackageVersion(0x907da0, 0xc0002ff020, 0xc0000c6a20, 0xc0004de04f, 0x5, 0xc000353e0b, 0x12)
	/home/runner/work/tools/tools/cmd/autoupdate/main.go:211 +0x2fb
main.main()
	/home/runner/work/tools/tools/cmd/autoupdate/main.go:91 +0x4ee
```
This is probably casued by: https://github.com/cdnjs/tools/pull/34

--- a/cmd/autoupdate/main.go
+++ b/cmd/autoupdate/main.go
@@ -87,7 +87,9 @@ func main() {
 
 		if !noUpdate && len(newVersionsToCommit) > 0 {
 			commitNewVersions(ctx, newVersionsToCommit)
-			commitPackageVersion(ctx, pckg, latestVersion, f)
+			if *pckg.Version != latestVersion {
+				commitPackageVersion(ctx, pckg, latestVersion, f)
+			}
 		}
 	}
 

